data/Field.js

```diff
@@ -88,6 +88,13 @@ export function parseFieldValue(val, type, defaultValue = null, disableXssProtec
 
     const FT = FieldType;
```

Comment:
just looking at this, and it occurs to me that we might be sanitizing *twice* if we send down a single string and cast it to an array.  Even worse, I think we are actually sanitizing string representations of dates/local dates/booleans, for no reason -- and we know this is expensive.  Should we have a method that bundles the sanitization + flag check and just conditionally call it  withing each block of the switch statement?

Reply:
Makes sense I think, since we only care about sanitizing values which would actually make it into record data (and then potentially back to a server), right?

Reply:
great!